Dockerfile

```diff
@@ -28,6 +33,9 @@ RUN set -ex; \
   yarn; \
   cp -p .env.example .env; \
   pipenv install --system --deploy; \
```

Comment:
Maybe consider using dockerfile's `ENV` command here to setup this environment variable, so something like
```
ENV DJANGO_SETTINGS_MODULE buza.settings_env
```
should do the trick